instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Post from "./Post";
+
+const PostContainer = props => {
+  return (
+    <div className="post-wrapper">
+      {props.posts.map(p => (
+        <Post key={p.imageUrl} post={p} />
```

Comment:
your use of single letter arguments does save typing out a full word but it is usually considered bad code style to do this except in legacy code, but then that was used because mathematicians did not like to write anything other than a few squiggles on a chalk board. so it derived from those times. the use of more verbose variable names has become a good practice code style and been quite useful for software engineering. your overall code style is impeccable but i would advise revising your choice of 1 letter variables.  (in c this would actually be convention but that stems from the mathematicians using it alot from its early revisions and has stuck since legacy) but in modern languages the convention for most code style guidelines require verbose variable and argument names